lib/runners/julia.js

```diff
@@ -17,12 +19,21 @@ module.exports.run = function run(opts, cb) {
             });
         },
         testIntegration: function (run) {
-            codeWriteSync('julia', opts.solution, juliaCodeDir);
+            codeWriteSync('julia', opts.code, juliaCodeDir, 'solution.jl', true);
+
             if (opts.setup) codeWriteSync('julia', opts.setup, juliaCodeDir);
```

Comment:
Note that this `opts.setup` is not used now because the entire `juliaCodeDir` is not being pulled in, since it didn't seem to be working as desired. Maybe you could add it to the solution which is written or to the `runCmd` OR you could get the dir reading thing to work again. It did say the -P arg was deprecated when I was looking at this which convinced me to remove it